<?xml version="1.0" encoding="UTF-8"?>
    <source package="javax.management.remote">
        <import package="java.io.IOException"/>
        <import package="javax.management.MBeanServer"/>
        <class name="JMXServerErrorException" line="34">
            <extends class="IOException"/>
            <javadoc line="34">
                Exception thrown as the result of a remote {@link MBeanServer}method invocation when an &lt;code&gt;Error&lt;/code&gt; is thrown while
                  processing the invocation in the remote MBean server.  A
                  &lt;code&gt;JMXServerErrorException&lt;/code&gt; instance contains the original
                  &lt;code&gt;Error&lt;/code&gt; that occurred as its cause.                
                <see>
                    java.rmi.ServerError                    
                </see>
                <since>
                    1.5                    
                </since>
            </javadoc>
            <declaration name="serialVersionUID" type="long" line="46"/>
            <javadoc line="48">
                Constructs a &lt;code&gt;JMXServerErrorException&lt;/code&gt; with the specified
                  detail message and nested error.                
                <param>
                    s the detail message.                    
                </param>
                <param>
                    err the nested error.  An instance of this class can be
                      constructed where this parameter is null, but the standard
                      connectors will never do so.                    
                </param>
            </javadoc>
            <method name="JMXServerErrorException" type="constructor" line="57">
                <params>
                    <param name="s" type="String"/>
                    <param name="err" type="Error"/>
                </params>
            </method>
            <method name="getCause" type="Throwable" line="62"/>
            <declaration name="cause" type="Error" line="66"/>
            <javadoc line="66">
                @serial An {@link Error} that caused this exception to be thrown.                
                <see>
                    #getCause()                    
                </see>
            </javadoc>
        </class>
    </source>